### DISCUSSION TITLE
Vite项目打包后使用qiankun访问时，样式丢失，如果项目单独运行没有任何问题

### DISCUSSION THREAD

**Discussion by DongZeMan** [umijs/qiankun#3024] [Q&A]
我项目是Vite+Vue3，base使用的绝对路径，打包之后，作为子应用访问，访问时样式全都失效了，如果不做为子应用，单独运行就没有问题，有大佬知道啥原因吗？怎么解决呢？

**Comment by NationalProgrammer6** [comment]
同样的问题，打包后的HTmL中Link引入的样式不生效。

**Comment by DongZeMan** [comment]
原因是主应用的head中没有link你子应用的css文件，需要自己手动去获取子应用index.html里的link标签然后插入到主应用的head标签中即可解决
